src/formatDate.js

```diff
@@ -51,6 +51,44 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const OLD_SEPARATOR = fromFormat[3];
```

Comment:
Consider using camelCase in the standard variables name

Please fix this in an all file